src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,102 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Handles the loading and saving of data in file,
+ * where data represents the list of tasks
+ * to be done
+ */
+
+
+public class Storage {
+    private static String filePath;
+    private static ArrayList<Task> tasks;
+    private Scanner reader;
+    private File file;
+
+
+    Storage(String filePath) {
+        
+        this.filePath = filePath;
+        this.file = new File(filePath);
+        this.tasks = new ArrayList<>();
```

Comment:
I don't think there's a need for the "this" on these 2 lines since the fields aren't shadowed by a constructor parameter? (Based on the advanced conventions)